lib/components/src/Zoom/Zoom.tsx

```diff
@@ -1,19 +1,6 @@
-import global from 'global';
 import { ZoomElement as Element } from './ZoomElement';
 import { ZoomIFrame as IFrame } from './ZoomIFrame';
 
-const { window: globalWindow } = global;
-
-export const browserSupportsCssZoom = (): boolean => {
```

Comment:
This function is never called and it looks like it moved here:

https://github.com/storybookjs/storybook/blob/a8db9585f07d8cfc7354e68f6b780a6ff6536b38/lib/components/src/Zoom/browserSupportsCssZoom.ts#L5